Scripts/ubuntu.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+command -v docker &> /dev/null  || { echo >&2 "Install docker https://www.docker.com"; exit 1; }
+ACTION=$1
+SWIFT=$2
+UBUNTU=$3
+DOCKERFILE=$(mktemp)
+echo "FROM swift:$SWIFT-$UBUNTU"     >> $DOCKERFILE
```

Comment:
```suggestion
echo "FROM swift:$SWIFT-$UBUNTU"     > $DOCKERFILE
```

Probably want the first to truncate?